### PR TITLE
Improve awesome-discord to align with Sindre's requests

### PR DESCRIPTION
**Pull request by justdotJS**
## Improve README.md
- Start descriptions with capital letter
- End descriptions with period
- Change URLS to Links
- Remove Discord Nitro as it doesn't fit with rest of Links

## New CONTRIBUTING.md
- Use Sindre's awesome-nodejs as a template
- Adapt template for awesome-discord

## New CODE-OF-CONDUCT.md
- Use Contributor Covenant Template
- Request contact on Discord

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -26,25 +26,25 @@
...
 - [PvPCraft](https://bot.pvpcraft.ca) - Multipurpose bot.
-- [Dank Memer](https://discordapp.com/oauth2/authorize?client_id=270904126974590976&scope=bot&permissions=3533888) - Best meme bot for Discord. No other functions.
+- [Dank Memer](https://discordapp.com/oauth2/authorize?client_id=270904126974590976&scope=bot&permissions=3533888) - Best meme bot for Discord, no other functions.
 - [OverwatchBot](https://discordapp.com/login?redirect_to=%2Foauth2%2Fauthorize%3F%26client_id%3D188064750351941633%26scope%3Dbot) - Retrieve Overwatch game statistics.
-- [Nadeko](https://discordapp.com/oauth2/authorize?client_id=170254782546575360&scope=bot&permissions=66186303) - Multipurpose bot.
+- [Nadeko](https://discordapp.com/oauth2/authorize?client_id=170254782546575360&scope=bot&permissions=66186303) - Popular multipurpose bot.
 - [NotSoBot](https://discordapp.com/login?redirect_to=%2Foauth2%2Fauthorize%3Fclient_id%3D170903265565736960%26scope%3Dbot) - Bot for image manipulation and meme making.
-- [DSL](https://discordservers.com) - Bot that places your server on DiscordServers.com
+- [DSL](https://discordservers.com) - Bot that places your server on DiscordServers.
```
Comment (byparallax) [inline readme.md]: The .com is a part of their brand identity and must stay here.

File: readme.md [hunk 2]
```diff
@@ -128,9 +128,8 @@
   - [Discord API](https://discord.gg/api)
   - [Discord Feedback](https://discord.gg/discord-feedback)
   - [Discord Events](https://discord.gg/events)
-- URLS
+- Links
   - [Developer Documentation](https://discordapp.com/developers)
-  - [Discord Nitro](https://discordapp.com/nitro)
```
Comment (byparallax) [inline readme.md]: Discord Nitro is important and relevant, it should stay here.

Comment (byparallax) [inline readme.md]: Discord Nitro is important and relevant, it should stay here.

File: readme.md (hunk 1 above)
Comment (byparallax) [inline readme.md]: The .com is a part of their brand identity and must stay here.

**Comment by justdotJS** [comment]
OK, I added the .com back, as well as Discord Nitro. However, as per @sindresorhus, I also added a description of Discord Nitro.

**Comment by byparallax** [comment]
Alright, I got confused for a few minutes with Github's interface. I am merging this PR.

File: readme.md (hunk 2 above)
Comment (justdotJS) [reply readme.md]: Already fixed in my latest commit.

**Pull request merged**